Formula/zbar.rb

```diff
@@ -21,12 +39,13 @@ class Zbar < Formula
   depends_on "freetype"
   depends_on "libtool" => :run
 
-  # Fix JPEG handling using patch from
-  # https://sourceforge.net/p/zbar/discussion/664596/thread/58b8d79b#8f67
-  # already applied upstream but not present in the 0.10 release
-  patch :DATA
-
   def install
+    if build.head?
+      inreplace "configure.ac", "-Werror", ""
```

Comment:
The upstream project is rather defunct and hasn't been touched since 2012. [This issue has been brought up on the mailing list](https://sourceforge.net/p/zbar/discussion/664596/thread/f4b52988/) back in 2012, so they should be aware of it.

The reason we need it is because without it, `autoreconf` ends up failing because:
```
automake: warnings are treated as errors
gtk/Makefile.am.inc:13: warning: '%'-style pattern rules are a GNU make extension
Makefile.am:31:   'gtk/Makefile.am.inc' included from here
gtk/Makefile.am.inc:17: warning: '%'-style pattern rules are a GNU make extension
Makefile.am:31:   'gtk/Makefile.am.inc' included from here
pygtk/Makefile.am.inc:20: warning: '%'-style pattern rules are a GNU make extension
Makefile.am:34:   'pygtk/Makefile.am.inc' included from here
...etc...
```